deployment/7_deploy_2.5_infrastructure.js

```diff
@@ -114,20 +119,54 @@ const main = async () => {
       config.defi.paraswap.adapters.linkswap || ethers.constants.AddressZero,
       config.defi.paraswap.adapters.defiswap || ethers.constants.AddressZero,
       config.defi.paraswap.adapters.zeroexV2 || ethers.constants.AddressZero,
-      config.defi.paraswap.adapters.zeroexV4 || ethers.constants.AddressZero
+      config.defi.paraswap.adapters.zeroexV4 || ethers.constants.AddressZero,
+      config.defi.paraswap.adapters.curve || ethers.constants.AddressZero
     ],
-    config.defi.paraswap.targetExchanges || [],
+    Object.values(config.defi.paraswap.targetExchanges || {}),
     config.defi.paraswap.marketMakers || [],
   );
   console.log(`Deployed ParaswapFilter at ${ParaswapFilterWrapper.address}`);
   await DappRegistryWrapper.addDapp(0, config.defi.paraswap.contract, ParaswapFilterWrapper.address);
 
+  console.log("Deploying ParaswapUniV2RouterFilter");
+  const factories = [config.defi.uniswap.factoryV2, ...config.defi.paraswap.uniswapForks.map((f) => f.factory)];
+  const initCodes = [config.defi.uniswap.initCodeV2, ...config.defi.paraswap.uniswapForks.map((f) => f.initCode)];
+  const routers = [config.defi.uniswap.paraswapUniV2Router, ...config.defi.paraswap.uniswapForks.map((f) => f.paraswapUniV2Router)];
+  for (let i = 0; i < routers.length; i += 1) {
+    const ParaswapUniV2RouterFilterWrapper = await ParaswapUniV2RouterFilter.new(
+      TokenRegistryWrapper.address,
+      factories[i],
+      initCodes[i],
+      config.defi.weth
+    );
+    console.log(`Deployed ParaswapUniV2RouterFilter #${i} at ${ParaswapUniV2RouterFilterWrapper.address}`);
+    await DappRegistryWrapper.addDapp(0, routers[i], ParaswapUniV2RouterFilterWrapper.address);
+  }
+
   // Paraswap Proxy
   console.log("Deploying OnlyApproveFilter");
   const OnlyApproveFilterWrapper = await OnlyApproveFilter.new();
   console.log(`Deployed OnlyApproveFilter at ${OnlyApproveFilterWrapper.address}`);
   const AugustusSwapperWrapper = await IAugustusSwapper.at(config.defi.paraswap.contract);
-  await DappRegistryWrapper.addDapp(0, await AugustusSwapperWrapper.getTokenTransferProxy(), OnlyApproveFilterWrapper.address);
+  const proxies = [await AugustusSwapperWrapper.getTokenTransferProxy(), ...Object.values(config.defi.paraswap.proxies || {})];
```

Comment:
Did you add the ZeroxV2 asset proxy to the config?

Reply:
Done